### ISSUE TITLE
Run tests on all supported Python version

### ISSUE THREAD

**Issue by abey79** [rerun-io/rerun#7510]
To the best of my knowledge, everything is done with 3.11 on CI, because that's what's configured in `pixi.toml`. However, we claim to support 3.8 and up, so we should test at least that.

This is 90% solved by:
- https://github.com/rerun-io/rerun/issues/7843

**Comment by abey79** [comment]
We used to have `nox` for that (but didn't actually use it). No longer: https://github.com/rerun-io/rerun/issues/5898.

**Comment by Wumpf** [comment]
Related to
* https://github.com/rerun-io/rerun/issues/2254
* https://github.com/rerun-io/rerun/issues/5525

**Comment by abey79** [comment]
I dont think this makes sense. https://github.com/rerun-io/rerun/issues/7843 is enough.
